src/SouffleInterface.h

```diff
@@ -376,6 +376,51 @@ class Relation {
      * in the table, set the next element pointer points to the current element itself.
      */
     virtual void purge() = 0;
+
+    /**
+     * Helper function for the wrapper function
+     */
+    template <typename Tuple, size_t N>
+    struct tuple_insert {
+        static void add(const Tuple& t, souffle::tuple& t1) {
+            tuple_insert<Tuple, N - 1>::add(t, t1);
+            t1 << std::get<N - 1>(t);
+        }
+    };
+
+    /**
+     * Helper function for the wrapper function in the first element
```

Comment:
State which wrapper function this is a helper function for, also say "for the first element" and not "in the first element".